ansible/roles/kamailio/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# Kamailio version
+kamailio_major_version: 5
+kamailio_minor_version: 5
+
+# Services host IP addresses
+gwlauncher_host: 192.168.92.1
+coturn_host: 192.168.92.3
```

Comment:
The `gwlauncher_host` and  `coturn_host` variables should be placed in the `defaults` folder